shared/chat/conversation/info-panel/index.js

```diff
@@ -26,6 +26,14 @@ const listStyle = {
       }),
 }
 
+let keyState = 0
```

Comment:
This is a bit weird since then `render` returns different stuff every time you call it. I think it's better to move this variable/function inside of `render()`, so you can number the dividers consistently.